plugins/woocommerce-blocks/tests/e2e/tests/product-collection/register-product-collection.block_theme.spec.ts

```diff
@@ -73,15 +74,11 @@ test.describe( 'Product Collection: Register Product Collection', () => {
 			} )
 			.click();
 
-		const productCollectionBlock = editor.canvas.getByLabel(
-			'Block: Product Collection'
-		);
-
 		for ( const myCollection of Object.values(
 			MY_REGISTERED_COLLECTIONS
 		) ) {
 			await expect(
-				productCollectionBlock.getByRole( 'button', {
+				page.getByRole( 'button', {
```

Comment:
Collections are displayed in `components-popover`. It used to be available from canvas but apparently the way it is rendered or place it is anchored changed and should be looked for on `page`.